### PR TITLE
Update assist docs

### PR DESCRIPTION
**Pull request by klizhentas**

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/ai-assist.mdx [hunk 1]
```diff
@@ -83,20 +83,20 @@ Do not share your secret API keys in publicly accessible areas.
 ## Step 2/3. Enable Assist
 
 To enable Teleport Assist, you need to provide your OpenAI API key. On each
-Proxy Service host, perform the following actions:
+Auth Service host, perform the following actions:
```
Comment (jakule) [inline docs/pages/ai-assist.mdx:86]: This is not exactly true. Right now OpenAI key is required in Auth and Proxy.

Comment (klizhentas) [reply docs/pages/ai-assist.mdx:86]: so you need it in both `auth_service` and `proxy_service`?

Comment (jakule) [reply docs/pages/ai-assist.mdx]: Yes, added to this PR.

**Review by jakule — APPROVED** [review]

**Comment by jakule** [comment]
@ptgott @alexfornuto Friendly ping

File: docs/pages/ai-assist.mdx [hunk 2]
```diff
@@ -83,20 +83,31 @@ Do not share your secret API keys in publicly accessible areas.
...
+If the host is running Auth service, add the following section:
+
  ```yaml
+auth_service:
+  assist:
+    openai:
+      api_token_path: /etc/teleport/openai_key
+```
+
+If the host is running Proxy service, add the following section:
```
Comment (ptgott) [inline docs/pages/ai-assist.mdx:100]:
```suggestion
If the host is running the Proxy Service, add the following section:
```

**Review by ptgott — APPROVED** [review]
Approved with minor suggestions

**Pull request merged**